markdown/h2m/handlers/index.ts

```diff
@@ -307,4 +311,6 @@ export const handlers: QueryAndTransform[] = [
   ),
 
   ["var", (node, t) => h("emphasis", t(node))],
+  ["dfn", (node) => h("inlineCode", toText(node))],
```

Comment:
Eeek, I made a mistake. `dfn` should map to `em`, not `code`. Sorry!

Reply:
No problemo, fixed!